thrust/thrust/tuple.h

```diff
@@ -39,6 +39,8 @@
 #  pragma system_header
 #endif // no system header
 
+#include <thrust/type_traits/is_trivially_relocatable.h>
```

Comment:
I am of the opinion that we should be moving this trait into libcu++ and derive it from `is_trivially_copyable` as a default and then users to specialize it.

But that may be something for another day. Note that the definition from the paper is much more complex and would require compiler support

Reply:
I agree. I also check for this trait inside CUB. I don't mind how it's called and whether it is really about `relocatability`. It should just be `is_trivially_copyable` with opt-in.